adafruit_pn532/adafruit_pn532.py

```diff
@@ -300,7 +317,9 @@ def send_command(
             raise RuntimeError("Did not receive expected ACK from PN532!")
         return True
 
-    def process_response(self, command, response_length=0, timeout=1):
+    def process_response(
+        self, command: int, response_length: int = 0, timeout: float = 1
+    ) -> bytearray:
```

Comment:
This can return `None` and can return `Union[bytes, bytearray]`, so the return should be `Optional[Union[bytes, bytearray]]`.